### PR TITLE
#3398 

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/3398 - [studio-ui] Publish dialog: Submit button should be disabled if nothing is selected. #3398

### CODE REVIEW

File: static-assets/components/cstudio-view-controllers/approve.js [hunk 1]
```diff
@@ -21,18 +21,18 @@
...
-(function(CStudioAuthoring){
+(function (CStudioAuthoring) {
 
     var Base = CStudioAuthoring.ViewController.Base,
         Dom = YAHOO.util.Dom,
         Event = YAHOO.util.Event,
         each = CStudioAuthoring.Utils.each,
         genDependency = [];
-        $ = jQuery;
+    $ = jQuery;
```
Comment (rart) [inline static-assets/components/cstudio-view-controllers/approve.js]: Declaring global variable.

File: static-assets/components/cstudio-view-controllers/approve.js [hunk 2]
```diff
@@ -142,12 +141,12 @@
             if (!allHaveSameDate) {
                 break;
             }
-/*
-            if (children.length > 0) {
-                allHaveSameDate = traverse(children, referenceDate);
-            }
-
-*/
+            /*
```
Comment (rart) [inline static-assets/components/cstudio-view-controllers/approve.js]: Remove comment

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/components/cstudio-view-controllers/approve.js (hunk 1 above)
Comment (vestrada) [reply static-assets/components/cstudio-view-controllers/approve.js]: Done

File: static-assets/components/cstudio-view-controllers/approve.js (hunk 2 above)
Comment (vestrada) [reply static-assets/components/cstudio-view-controllers/approve.js]: Done

**Review by rart — APPROVED** [review]

**Pull request merged**